### PR TITLE
[Issue 5142][docs] Remove non-partitioned topic info from partitioned topic docs

### LINKED ISSUE #5142: [docs] Managing partitioned topics page has odd "Nonpartitioned topics resources" section
**Describe the bug**

The docs about managing partitioned topics has an odd sub-heading "Nonpartitioned topics resources". However, everything under that sub-heading except the first "Create" sub-heading seems to be related to partitioned topics (as one would expect for this page). The "Nonpartitioned topics resources" and the "Create" subheading under that seem to be a copy/paste error or something.

image

### PR DESCRIPTION
**Pull request by rocketraman**
Fixes #5142

### Motivation

Docs are confusing.

### Modifications

Edited out the incorrect section in the docs. Moved section into its documentation page, with additional related docs.

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

### CODE REVIEW

**Comment by Jennifer88huang** [comment]
The information is placed wrong. We can delete it from the current place.
I have a suggestion on this: while we delete the content from this place, we need to add more information. See suggestion at https://github.com/apache/pulsar/issues/5142

File: site2/docs/admin-api-non-partitioned-topics.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+#### REST API
+
+{@inject: endpoint|PUT|/admin/v2/persistent/:tenant/:namespace/:topic|operation/createNonPartitionedTopic}
+
+#### Java
+
+```java
+String topicName = "persistent://my-tenant/my-namespace/my-topic";
+// this does not seem to exist?
+admin.topics().create(topicName);
```
Comment (rocketraman) [inline site2/docs/admin-api-non-partitioned-topics.md]: @Jennifer88huang @sijie Note this, I could not find this method in the actual API docs https://pulsar.apache.org/api/admin/org/apache/pulsar/client/admin/Topics.html. Not sure why.

**Comment by Jennifer88huang** [comment]
@zymap Could you please help check whether the commands and usage for the non-partitioned topics are right? Thank you very much.

**Comment by Jennifer88huang** [comment]
@rocketraman When we add a new topic, remember to add it also in the sidebars.json file.

File: site2/docs/admin-api-non-partitioned-topics.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+#### Java
+
+```java
+admin.topics().delete(persistentTopic);
+```
+
+### List
+
+It provides a list of persistent topics existing under a given namespace.
```
Comment (zymap) [inline site2/docs/admin-api-non-partitioned-topics.md]:
```suggestion
It provides a list of topics existing under a given namespace.  
```

File: site2/docs/admin-api-non-partitioned-topics.md (hunk 1 above)
Comment (sijie) [reply site2/docs/admin-api-non-partitioned-topics.md]: I think it should be `createNonPartitionedTopic`: https://github.com/apache/pulsar/blob/master/pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/Topics.java#L203

Although I am not sure why javadoc is not generated correctly, we need to lookup into the javadoc build part.

Comment (tuteng) [reply site2/docs/admin-api-non-partitioned-topics.md]: https://github.com/apache/pulsar/issues/4979

**Comment by Jennifer88huang** [comment]
@rocketraman Well done! Thanks for your efforts.

**Comment by sijie** [comment]
run cpp tests

**Pull request merged**